Siri Shortuct Handler/SiriHandlerViewModel.swift

```diff
@@ -116,6 +116,32 @@ class SiriHandlerViewModel {
         }
     }
     
+    public func getConnectionStatus(_ completion: @escaping (GetConnectionStatusIntentResponse) -> Void) {
+        guard let vpnGateway = vpnGateway else {
```

Comment:
I would suggest returning "Not logged in" string in `getConnectionStatusString` instead of opening the app. Could you please make those changes?

Reply:
There are two existing localized strings that I think would work instead of adding a new one:

- openAppToLogIn
- logIn ("_log_in")

Or would you prefer "Not logged in" exactly? Thoughts?

Reply:
I think if Siri will answer `ProtonVPN Says: "Log in"` it may be not be enough for user to understand what's going on. So I would suggest adding separate translation string (like `_vpnstatus_not_loggedin` with a text `You are not logged in to ProtonVPN. Open the ProtonVPN app and sign in.`).

Reply:
Thank You for contributing, you code will be merged with the next release.

Reply:
Thank you for being responsive and accepting community contributions.